demo/settings.py

```diff
@@ -102,3 +102,5 @@
         "ROUTING": "grad_cam.routing.channel_routing",
     },
 }
+
+X_FRAME_OPTIONS = 'https://cloudcv.org/projects/grad-cam'
```

Comment:
I am not sure if this is going to work. Can you please point me to the relevant documentation for the same. Also, it would be nice to put some description in the PR. :)

Reply:
Here is the documentation: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options